web/modules/custom/joinup_community_content/js/pinned_content.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @file
+ * Attaches the behaviours to show visual cues on pinned community content.
+ */
+
+(function ($, Drupal) {
+
+  /**
+   * Theme function that renders a visual cue for pinned content.
+   *
+   * @returns {string}
+   *   The HTML for the pinned content visual cue.
+   */
+  Drupal.theme.pinnedContentCue = function () {
+    return '<div class="pinned-markup"><div class="icon icon--pin"></div></div>';
```

Comment:
A node can be pinned only inside its collection, so there is no need to have an explicative tooltip.